templates/ingress.yaml

```diff
@@ -381,5 +393,32 @@ spec:
           serviceName: envproxy
           servicePort: 8080
 {{- end }}
----
+    {{- else }}
+      - path: /proxy/*
+        backend:
+          serviceName: envproxy
+          servicePort: 8080
+      - path: /api/*
+        backend:
+          serviceName: cemanager
+          servicePort: 8080
+      - path: /auth/*
+        backend:
+          serviceName: cemanager
+          servicePort: 8080
+      - path: /*
+        backend:
+          serviceName: dashboard
+          servicePort: 3000
+{{- if ne .Values.devurls.host "" }}
+  - host: {{ .Values.devurls.host | quote }}
+    http:
+      paths:
+      - path: /*
+        backend:
+          serviceName: envproxy
+          servicePort: 8080
 {{- end }}
+    {{- end }}
+{{- end }}
```

Comment:
the indentations for the `{{ if` are tripping me out. Is it possible to format it better?

Reply:
kinda awkward cuz i tried to keep them at the indentation of what they wrapped. fixed tho